packages/plugins/test/mocha/cli/propose.spec.ts

```diff
@@ -715,6 +716,93 @@ describe('Propose CLI command', () => {
 
     expect(errorThrown).to.be.true
   })
+
+  describe('Issues', () => {
+    it('CHU-676: Deploys with call to safeAddress() in script', async () => {
```

Comment:
Couple thoughts here: 
1. We should start following a pattern where we label which tests are related to which issues, ideally, we would also organize regression tests together into suites. I've tried to start that pattern here. The Foundry devs do this, and it's really nice for keeping things organized. It also makes it easy to find additional context on specific tests, which is super useful if you ever make a change that breaks them. 

2. I implemented the regression test for this issue as a full-on integration test. This is less than ideal for regression tests like this b/c we will eventually have a ton of them. We should generally try to avoid doing this, especially if the test is specific to the Solidity library contracts. Ideally, the test for this would have been a simple Foundry test that calls the sphinxCollect function and checks the results. I put what I would consider a reasonable amount of effort into trying to do that and found it impractical in this case.

Reply:
> We should start following a pattern where we label which tests are related to which issues, ideally, we would also organize regression tests together into suites. I've tried to start that pattern here. The Foundry devs do this, and it's really nice for keeping things organized. It also makes it easy to find additional context on specific tests, which is super useful if you ever make a change that breaks them.

i like the idea of labeling tests with tickets, but i think i need clarity on how strictly we want to adhere to this pattern. for example, say the easiest way to implement a regression test is to add slightly more coverage to an existing test case, or to an existing helper function that a lot of test cases use (like `assertValidProposalRequest`). should we make an entirely different test case in this scenario? i think this deserves its own discussion, so i made a note to talk about it on fri